js/src/jsonrpc/generator/build-markdown.js

```diff
@@ -16,25 +16,46 @@
 
 import fs from 'fs';
 import path from 'path';
+import chalk from 'chalk';
 
+import { DUMMY } from '../helpers';
+import { BlockNumber } from '../types';
 import interfaces from '../';
 
-const MARKDOWN = path.join(__dirname, '../../release/interfaces.md');
+const ROOT_DIR = path.join(__dirname, '../docs');
 
-let preamble = '# interfaces\n';
-let markdown = '';
+if (!fs.existsSync(ROOT_DIR)) {
+  fs.mkdirSync(ROOT_DIR);
+}
+
+// Logging helpers
+function info (log) { console.log(chalk.blue(`INFO:\t${log}`)); }
```

Comment:
Small comment, which would be more applicable to code not for the generator (i.e. these scripts I probably expect to always be a bit more messy - just the nature of the beast.) Would generally prefer even the simple functions to have the format of

```
function info (log) { 
   console.log(chalk.blue(`INFO:\t${log}`));
}
```

In addition blank line between. (non-critical here, nor a showstopper)